api/rs/slint/Cargo.toml

```diff
@@ -38,13 +38,16 @@ std = ["i-slint-core/std"]
 backend-qt = ["i-slint-backend-selector/i-slint-backend-qt", "std"]
 
 ## The GL backend uses the `winit` crate for the windowing system integration,
-## and the `femtovg` crate for the rendering.
-## On unix, at least one of `x11` or `wayland` features must be enabled.
-backend-gl = ["i-slint-backend-selector/i-slint-backend-gl", "std"]
-## Same as the GL backend, with the `x11` feature of `winit` enabled
-x11 = ["i-slint-backend-selector/x11", "backend-gl"]
-## Same as the GL backend, with the `wayland` feature of `winit` enabled
-wayland = ["i-slint-backend-selector/wayland", "backend-gl"]
+## and the `femtovg` crate for the rendering. With this feature, all windowing
+## systems are supported. For a smaller build, omit this feature and select
+## one of the other specific `backend-gl-XX` features.
+backend-gl-all = ["i-slint-backend-selector/backend-gl-all", "std"]
+## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
+## X Window System.
+backend-gl-x11 = ["i-slint-backend-selector/backend-gl-x11", "std"]
+## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
+## Wayland window system.
+backend-gl-wayland = ["i-slint-backend-selector/backend-gl-wayland", "std"]
```

Comment:
```suggestion
backend-gl-all = ["i-slint-backend-selector/backend-gl-all", "std"]
## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
## X Window System on Unix.
backend-gl-x11 = ["i-slint-backend-selector/backend-gl-x11", "std"]
## Simliar to `backend-gl-all` this enables the GL backend but only with support for the
## Wayland window system on Unix.
backend-gl-wayland = ["i-slint-backend-selector/backend-gl-wayland", "std"]
```

tying to convey that the support also works on other platform

Reply:
Thanks, good idea. I'll apply this to all three places (rs, interpreter and CMake).